Commands/Taxonomy/NewTermGroup.cs

```diff
@@ -45,7 +49,7 @@ protected override void ExecuteCmdlet()
             }
             // Create Group
             var group = termStore.CreateTermGroup(GroupName, GroupId, Description);
-
+            ClientContext.ExecuteQueryRetry();
```

Comment:
This ExecuteQueryRetry is not needed as we execute the query in the extension method already.